ci/redhat-yum/apache-apisix.repo

```diff
@@ -0,0 +1,25 @@
+#
```

Comment:
Why we need this?

Reply:
In all other files, license started from second line so I left the first line here as well following convention.

Reply:
I mean, why we need this file？

Reply:
Yes you're right, we don't need it anymore. In previous commit, I was using it. I removed the file